README.md

```diff
@@ -1,8 +1,9 @@
-# altinn-design-system
-
-Design system v2  
+# Storybook for Altinn
 [designsystem.altinn.studio/](https://designsystem.altinn.studio/)
 
+⚠️ This storybook is used for components that are unique for Altinn. We have moved reusable core components that can be used in multiple products and situations to our [common design system](https://digdir.github.io/designsystem/?path=/story/introduksjon--page). We will continue to develop reusable components in the [common design system](https://digdir.github.io/designsystem/?path=/story/introduksjon--page).
```

Comment:
Storybook er bare et verktøy vi bruker for dokumentasjon, komponentene bor i designsystemet uavhengig av dette.
```suggestion
⚠️ This library is used for components that are unique for Altinn. We have moved reusable core components that can be used in multiple products and situations to our [common design system](https://digdir.github.io/designsystem/?path=/story/introduksjon--page). We will continue to develop reusable components in the [common design system](https://digdir.github.io/designsystem/?path=/story/introduksjon--page).
```

Reply:
Enig, takk!